ui/cypress/integration/error-pages.spec.js

```diff
@@ -0,0 +1,75 @@
+describe('Error Pages Navigation', () => {
+  beforeEach(() => {
+    cy.setupMocks();
+    cy.login();
+  });
+
+  it('brings me to 404 error page', () => {
+    cy.visit('/404');
+    cy.stubHistory();
+
+    cy.get('[data-cy="sc-error-page404"]').should('have.length', 1);
+  });
+
+  it('redirects me to 404 error page if route not found', () => {
+    cy.visit('/undefined-route-that-doesnt-exists');
+    cy.stubHistory();
+
+    cy.get('[data-cy="sc-error-page404"]').should('have.length', 1);
+  });
+
+  it('brings me to 500 error page', () => {
+    cy.visit('/500');
+    cy.stubHistory();
+
+    cy.get('[data-cy="sc-error-page500"]').should('have.length', 1);
+  });
+});
+
+describe('Error Pages Custom config', () => {
+  beforeEach(() => {
+    cy.setupMocks(null);
+    cy.login();
+  });
+
+  it('redirects me to 500 - fetch config results in internal error', () => {
+    cy.route2('GET', '/config.json', {
+      fixture: 'config.json',
+      statusCode: 500,
+    });
+    cy.visit('/');
+    cy.stubHistory();
+
+    cy.get('[data-cy="sc-error-page500"]').should('have.length', 1);
+  });
+
+  it('500 error page should contains support link if set to config.json', () => {
+    let url_support = 'https://mysupportlink';
+    cy.fixture('config.json').then((config) => {
+      config['url_support'] = url_support;
+      cy.route2('GET', '/config.json', config);
+    });
+
+    cy.visit('/500');
+    cy.stubHistory();
+
+    cy.get('[data-cy="sc-error-page500"]').should('have.length', 1);
+    cy.get('.sc-error-page500 * a')
+      .should('have.attr', 'href')
+      .and('include', url_support);
+  });
+});
```

Comment:
For me, this should not be an independent scenario, should be an assertion in a scenario. For example, when we test the navbar failing scenario, we can check if the support link is rendered on the 500 error page.